src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodRepository.scala

```diff
@@ -50,10 +50,15 @@ object MethodRepository {
     def toShortString: String = s"Method($namespace,$name,$snapshotId)"
   }
 
+  case class AgoraConfigurationShort(
```

Comment:
This file is full of stuff that's vaguely copy-pasta from Rawls

I'm not thrilled about adding to it, but I found that replacing usages of `MethodConfigurationShort` with `AgoraConfigurationShort` is a lot cleaner than doing

```
config.methodRepoMethod match {
  case agoraMethod: AgoraMethod => [...]
  case _ => throw Exception(...)
}
```
in a bunch of places. I'm not married to this solution though, if someone can find a nicer way to encode this information in the type system I'm glad to hear it.

Reply:
👍